internal/pkg/config/v1/config.go

```diff
@@ -15,13 +17,15 @@ import (
 	"github.com/confluentinc/cli/internal/pkg/config"
 	"github.com/confluentinc/cli/internal/pkg/errors"
 	"github.com/confluentinc/cli/internal/pkg/log"
+	"github.com/confluentinc/cli/internal/pkg/secret"
 	"github.com/confluentinc/cli/internal/pkg/utils"
 	pversion "github.com/confluentinc/cli/internal/pkg/version"
 )
 
 const (
 	defaultConfigFileFmt = "%s/.confluent/config.json"
 	emptyFieldIndicator  = "EMPTY"
+	tokenRegex           = `(^[\w-]*\.[\w-]*\.[\w-]*$)`
```

Comment:
I don't think we need the parentheses
```suggestion
	tokenRegex           = `^[\w-]*\.[\w-]*\.[\w-]*$`
```